src/hooks/useMarginRemStyle.ts

```diff
@@ -25,12 +30,18 @@ export interface MarginRemProps {
 
 export type MarginRemStyle = Pick<
   ViewStyle,
-  'marginTop' | 'marginBottom' | 'marginLeft' | 'marginRight'
+  'alignSelf' | 'marginTop' | 'marginBottom' | 'marginLeft' | 'marginRight'
 >
 
-export const useMarginRemStyle = (props: MarginRemProps): MarginRemStyle => {
+export const useMarginRemStyle = (
+  props: MarginRemProps,
+  // TODO: Remove this prop once all designs using this prop have been updated.
+  /** @deprecated Your design should expect the component to have 0.5rem margins */
+  defaultRem: number = 0.5
```

Comment:
This shouldn't be configurable, should just be a constant

Reply:
I know, it's only there to not break other places where useMarginRemStyle is used. See the comment.

Reply:
What about `{aroundRem: 0}`?

Reply:
That would override the defaultRem (should be called `fallbackRem` actually).

Reply:
I mean pass `{aroundRem: 0}` at the callsite if the `marignRem` props are not provided. Then we don't need to even have this deprecation TODO to deal with in the common hook.